site/docs/v1/tech/tutorials/integrate-dotnet.md

```diff
@@ -0,0 +1,240 @@
+
+After you’ve signed in, you’ll end up at the "Secure" page and will see all claims encoded in the JWT.
+
+{% include _image.liquid src="/assets/img/docs/integrations/dotnet-integration/dotnet-secure-page-claims.png" alt="Logged in page with claims" class="img-fluid bottom-cropped" width="1200" figure=false %}
+
+
+## Conclusion
+
+Your directory tree should look like this:
+
+```text
+├── docker-compose.yml
+├── SetupFusionauth
+│   ├── Program.cs
+│   ├── setup-fusionauth.csproj
+│   ├── setup-fusionauth.sln
+└── SetupDotnet
+    ├── appsettings.development.json
+    ├── appsettings.json
+    ├── bin/
+        ├── ...
+    ├── obj/
+        ├── ...
+    ├── Pages/
+    │   ├── _ViewImports.cshtml
+    │   ├── _ViewStart.cshtml
+    │   ├── Error.cshtml
+    │   ├── Error.cshtml.cs
+    │   ├── Index.cshtml
+    │   ├── Index.cshtml.cs
+    │   ├── Logout.cshtml
+    │   ├── Logout.cshtml.cs
+    │   ├── Privacy.cshtml
+    │   ├── Privacy.cshtml.cs
+    │   ├── Secure.cshtml
+    │   └── Secure.cshtml.cs
+    │   └── Shared/
+    │       ├── _Layout.cshtml
+    │       ├── _Layout.cshtml.css
+    │       └── _ValidateScriptsPartial.cshtml
+    ├── Program.cs
+    ├── Properties/
+    ├── SetupDotnet.csproj
+    ├── SetupDotnet.sln
+    ├── Startup.cs
+    └── wwwroot/
+```
+
+Once you’ve created this directory structure, you can start up the {{page.technology}} application using this command:
+
+```shell
+ASPNETCORE_ENVIRONMENT=Development SetupDotnet__ClientSecret='change-this-in-production-to-be-a-real-secret' bin/Debug/net7.0/osx-x64/publish/SetupDotnet
+```
+
+The full code is available [here](https://github.com/ritza-co/fusionauth-dotnet-integration).
+
+## Troubleshooting
+
+If you run into an issue with cookies on Chrome or other browsers, you might need to run the ASP.NET application under SSL.
+
+You can follow [this guide](https://learn.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide) to install development SSL certificates for your .NET environment.
+
+Alternatively, you can run the project using [Visual Studio](https://visualstudio.microsoft.com), which will run the project using SSL.
+
+If you do this, make sure to update the <span class="field">Authorized Redirect URL</span> to reflect the `https` protocol. Also note that the project will probably run on a different port when using SSL, so you must update that as well. To do so, log into the administrative user interface, navigate to <span class="breadcrumb">Applications</span>, then click the <span class="uielement">Edit</span> button on your application and navigate to the <span class="breadcrumb">OAuth</span> tab. You can have more than one URL.
+
+This tutorial has example versions built for a few versions of ASP.NET. View the following repos for the full code for various versions:
```

Comment:
Let's remove these, as they aren't actively maintained. I will archive these before merging this.